### ISSUE TITLE
[APP]

### ISSUE THREAD

**Issue by TimSkells** [PipedreamHQ/pipedream#3509]
Clockify and Connecteam?

**Comment by dylburger** [comment]
Thanks for the request! We've added this to [our integrations backlog](https://github.com/orgs/PipedreamHQ/projects/8/views/1), and we'll update this ticket with progress.

**Comment by sergioeliot2039** [comment]
Hi @YBLCompany , thanks for the request!

As of Clockify, this is already integrated, please check it out at https://pipedream.com/apps/clockify

For Connecteam, I have requested an API key to their Support

**Comment by sergioeliot2039** [comment]
Hey @TimSkells , I checked on your Connecteam app request, it looks like the API is available on Enterprise plan only.

As per their pricing page

> Enterprise
> A personal account manager, private branding, 2FA and biometric lock, API access, and more

Do you have API access yourself? If so could you please try adding it at https://pipedream.com/submit-an-app (this is in early Alpha, so let us know if you run into any issue), or otherwise do you have API Key you could share and revoke once the integration is done?

Btw sorry about the confusion earlier I tagged someone else

**Comment by dylburger** [comment]
Without more information, I'm going to close this issue. Feel free to reopen and add more details, if you'd like.

**Comment by dylburger** [comment]
Without more information, I'm going to close this issue. Feel free to reopen and add more details, if you'd like.

**Comment by sergioeliot2039** [comment]
I created a new issue to track separatedly Connectteam https://github.com/PipedreamHQ/pipedream/issues/4072

**Comment by dylburger** [comment]
Without more information, I'm going to close this issue. Feel free to reopen and add more details, if you'd like.
